src/test/scala/org/kibanaLoadTest/scenario/Canvas.scala

```diff
@@ -1,78 +1,126 @@
+    val defaultHeaders =
+      headers.combine(Map("Referer" -> s"$baseUrl/app/canvas"))
+    val headersWithXsrf = defaultHeaders.combine(
+      Map(
+        "kbn-xsrf" -> "professionally-crafted-string-of-text"
+      )
+    )
     exec(
-      http("canvas workpads")
-        .get("/api/canvas/workpad/find")
-        .queryParam("name", "")
-        .queryParam("perPage", "10000")
-        .headers(headers)
-        .header("Referer", baseUrl + "/app/canvas")
+      http("canvas: fns")
+        .get("/api/canvas/fns")
+        .headers(defaultHeaders)
         .check(status.is(200))
-        .check()
-        .check(jsonPath("$.workpads[0].id").saveAs("workpadId"))
-    ).exitBlockOnFail {
-      doIf(version.isAbove79x) {
-        exec(
-          http("interpreter demo")
-            .get(fnsPath)
-            .queryParam("name", "")
-            .queryParam("perPage", "10000")
-            .headers(headers)
-            .header("Referer", baseUrl + "/app/canvas")
-            .check(status.is(200))
-        ).pause(5)
-      }.exec(
-          http("load workpad")
-            .get("/api/canvas/workpad/${workpadId}")
-            .headers(headers)
-            .header("Referer", baseUrl + "/app/canvas")
-            .header("kbn-xsrf", "professionally-crafted-string-of-text")
-            .check(status.is(200))
-        )
-        .pause(1)
-        .exec(
-          http("query canvas timelion")
-            .post("/api/timelion/run")
-            .body(ElFileBody("data/canvasTimelionPayload.json"))
-            .asJson
-            .headers(headers)
-            .header("Referer", baseUrl + "/app/canvas")
-            .header("kbn-xsrf", "professionally-crafted-string-of-text")
-            .check(status.is(200))
-        )
-        .pause(1)
-        .exec(
-          http("query canvas aggs 1")
-            .post(fnsPath)
-            .body(ElFileBody("data/canvasInterpreterPayload1.json"))
-            .asJson
-            .headers(headers)
-            .header("Referer", baseUrl + "/app/canvas")
-            .check(status.is(200))
-        )
-        .pause(1)
-        .exec(
-          http("query canvas aggs 2")
-            .post(fnsPath)
-            .body(ElFileBody("data/canvasInterpreterPayload2.json"))
-            .asJson
-            .headers(headers)
-            .header("Referer", baseUrl + "/app/canvas")
-            .check(status.is(200))
-        )
-    }
+    ).exec(
+        http("canvas: load workpads list")
+          .get("/api/canvas/workpad/find")
+          .queryParam("name", "")
+          .queryParam("perPage", "10000")
+          .headers(defaultHeaders)
+          .check(status.is(200))
+          .check(jsonPath("$.workpads[0].id").saveAs("workpadId"))
+      )
+      .exec(session =>
+        session
+          .set("startMonth", Helper.getMonthStartDate())
+          .set("endMonth", Helper.getMonthStartDate(1))
+          .set("currentTime", Helper.getDate(Calendar.DAY_OF_MONTH, 0))
```

Comment:
When I load eCommerce workpad on Apr 13, I see 3 `/api/timelion/run` API calls with the following **time**:

- `from: "2020-04-13T12:45:06.695Z", to: "2021-04-13T12:45:06.695Z", interval: "1w", timezone: "UTC"`
- `{from: "2021-04-01T00:04:19.000Z", to: "2021-05-01T23:45:36.000Z", interval: "1d", timezone: "UTC"}`
- `{from: "2021-04-01T00:04:19.000Z", to: "2021-05-01T23:45:36.000Z", interval: "1d", timezone: "UTC"}` repeats the 2nd one

In load scenario I make first call with `current time`, 2nd - `{this_month_start_date, next_month_start_date}`. Does it replicate original behaviour or I miss smth?